report_aeroo/models/ir_actions_report.py

```diff
@@ -538,12 +545,30 @@ class AerooReportsWithAttachmentFilenamePerLang(models.Model):
     aeroo_filename_line_ids = fields.One2many(
         'aeroo.filename.line', 'report_id', 'Filenames by Language')
 
-    def _get_attachment_filename_template(self, record):
+    def get_aeroo_filename(self, record, output_format):
+        """Get the attachement filename for the generated report.
+
+        :param record: the record for which to generate the freport
+        :return: the filename
+        """
+        import ipdb;ipdb.set_trace()
```

Comment:
uhm, Are you sure you want that in production? 💣 😛

Reply:
ah, j'hésitait sur cette ligne, maintenant que tu en parles :smile: